src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1476,7 +1460,6 @@ private void ProcessAuthentication()
         /// </remarks>
         internal void SetRequestContent(HttpRequestMessage request, byte[] content)
         {
-            ArgumentNullException.ThrowIfNull(request);
```

Comment:
I wonder why this is removed. Please revert.
We could remove this after turn on nullable annotations.

Reply:
I removed it because SetRequestContent is always called from FillRequestStream that already throws if request is null. Was I wrong?

Reply:
It is internal API contract check. We prefer real check instead of Debug.Assert because we don't run tests in Debug mode.
After we enable nullable annotations compiler will do the work and we can safely remove most of internal null checks.